### PR TITLE
updated typo in paragraph

### PR DESCRIPTION
**Pull request by whoami-shubham**
replaced `too—` with `to` in this paragraph
<br/>
image

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -43,7 +43,7 @@
       </div>
       <div class="lg:col-2">
         <h3 class="text-lg">Build pipeline</h3>
-        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, too— build your app for production with a single command.</p>
+        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, to build your app for production with a single command.</p>
```
Comment (locks) [inline app/templates/index.hbs:46]:
```suggestion
        <p>Ember CLI is the backbone of modern Ember apps, providing code generators to create new entities and putting the necessary files in the right place, every time. Ember apps come with a <strong>built-in development environment</strong> with fast rebuilds, auto-reload, and a test runner! <br><br> Ready to deploy? That's easy, too! Build your app for production with a single command.</p>
```

**Review by locks — CHANGES_REQUESTED** [review]
I left a suggestion of what I think we should go with, what do you think?
The original is correct, but I agree that the emdash is very strange, especially with how the text breaks after the emdash.

File: app/templates/index.hbs (hunk 1 above)
Comment (whoami-shubham) [reply app/templates/index.hbs:46]: yes, that's why I thought ` too—` might be a typo. 
after replacing   `too—` with  `too!` it looks fine.

**Review by locks — APPROVED** [review]

**Comment by locks** [comment]
Appreciate the help :)

**Pull request merged**